revscores/datasources/site_info_doc.py

```diff
@@ -0,0 +1,7 @@
+from .datasource import Datasource
+
+def process(session):
+    
+        doc = session.site_info.query(properties={'namespaces'})
+        return doc['query']
+
```

Comment:
Shouldn' this file have a line similar to
`namespaces = Datasource("site_info_doc", process, ...)`
?